src/daemon/command_parser_executor.cpp

```diff
@@ -674,6 +674,29 @@ bool t_command_parser_executor::sync_info(const std::vector<std::string>& args)
   return m_executor.sync_info();
 }
 
+bool t_command_parser_executor::pop_blocks(const std::vector<std::string>& args)
+{
+  if (args.size() != 1) {
+    std::cout << "Exactly one parameter is needed" << std::endl;
+    return false;
+  }
+
+  try
+  {
+    uint64_t nblocks = boost::lexical_cast<uint64_t>(args[0]);
+    if (nblocks < 1) {
```

Comment:
Please keep the same brace style as the code you're modifying, and consistent within the patch. Most code has the brace on a new line, and half your code here also does.